content/cpp/concepts/vectors/vectors.md

```diff
@@ -68,6 +68,27 @@ Due to the dynamic nature of vectors, it can be hard to keep track of the number
 std::cout << order.size();
 ```
 
+### `.capacity()`
+
+`.capacity()` is used to retrieve the current capacity of a `std::vector`. The capacity of a vector represents the number of elements it can hold without needing to allocate additional memory. The vector may have more capacity than its current size to reduce the need for frequent reallocations and copying when elements are added. 
+Initially, the capacity is likely to be small, but as elements are added using `push_back()`, the capacity will grow to accommodate the added elements. The exact growth behavior may vary depending on the C++ implementation, but it's typically designed to minimize reallocations and copying.
+
+```cpp
+
+    std::vector<int> myVector;
```

Comment:
instead of creating a new vector for this function I would suggest to use and modify the vector "order" used in the other examples within this section